app/controllers/albumController.js

```diff
@@ -0,0 +1,14 @@
+const axios = require('axios'),
+  errors = require('../errors');
+
+exports.getAllAlbums = (req, res) => {
+  axios
```

Comment:
usually we try abstracting this kind of interactions inside a albumsService, so you can divide according to the responsability of each file and also reuse code more easily.